src/plugins/template/tests/functional/skip_tests_config.cpp

```diff
@@ -112,11 +112,23 @@ std::vector<std::string> disabledTestPatterns() {
         // CVS-95608
         R"(.*CachingSupportCase.*CompileModelCacheTestBase.*)",
         R"(.*OVClassLoadNetworkTest.*QueryNetworkMULTIWithHETERONoThrow_V10.*)",
+        // New plugin API doesn't support legacy NV12 I420 preprocessing
+        R"(.*ConvertNV12WithLegacyTest.*)",
+        R"(.*ConvertI420WithLegacyTest.*)",
+        // Plugin version was changed to ov::Version
+        R"(.*VersionTest.*pluginCurrentVersionIsCorrect.*)",
+        // New plugin doesn't support dynamic preprocessing, here we set blob with changed layout
+        R"(.*InferRequestPreprocessDynamicallyInSetBlobTest.*_iPRC=0.*_iLT=1.*)",
+        // New plugin API doesn't support changes of pre-processing
+        R"(.*InferRequestPreprocessTest.*SetPreProcessToInputInfo.*)",
+        R"(.*InferRequestPreprocessTest.*SetPreProcessToInferRequest.*)",
+        // TODO: Fix deep copy for inputs/output
+        R"(.*OVInferRequestBatchedTests.*SetInputTensors_Cache_CheckDeepCopy.*)",
```

Comment:
will we create a ticket for it?

Reply:
I will do it in this PR :)

Reply:
Enabled in the PR: https://github.com/openvinotoolkit/openvino/pull/15481